src/holisticai/robustness/attackers/regression/gb_attackers.py

```diff
@@ -270,3 +276,282 @@ def _comp_attack_trn(self, clf, wxc, bxc, wyc, byc, otherargs):  # noqa: ARG002
         attacky = np.dot(res, grady) / self.samplenum
 
         return attackx, attacky
+
+
+class RidgeGDPoisoner(GDPoisoner):
+    """
+    RidgeGDPoisoner implements computations for Ridge regression. Unlike LinRegGDPoisoner, RidgeGDPoisoner includes regularization terms in the computations.
+
+    Parameters
+    ----------
+    poison_proportion : float
+        The proportion of points to flip. Default is 0.2.
+    num_inits : int
+        The number of initializations. Default is 1.
+    max_iter : int
+        The maximum number of iterations. Default is 15.
+    initializer : str
+        The initialization method. Default is 'inf_flip'.
```

Comment:
tem que espeficicar que parametros podem ser de entrada

Reply:
Done. Added possible options for initializers